src/doc-structure.ts

```diff
@@ -139,9 +139,10 @@ export const docStructure: Record<string, DocsNavigation[]> = {
                     title: "Creating your first session",
                     path: "/first-session/",
                 },
+               
                 {
-                    title: "RTS-style lighting",
-                    path: "/rts-lighting/",
+                    title: "Wall Masks Guide",
```

Comment:
You've removed the rts style doc here

Reply:
Whoops must have happened when I reverted. My bad. I'll add it back.